projects/rocprofiler-sdk/source/lib/python/rocpd/summary.py

```diff
@@ -209,20 +227,27 @@ def generate_domain_query(connection: RocpdImportData, by_rank=False) -> Tuple[s
         join_condition = "CROSS JOIN total_duration TD"
         order_by = 'ORDER BY GD."DURATION (nsec)" DESC'
 
-    summary_views = [
-        itr for itr in get_temp_view_names(connection) if itr.endswith(view_suffix)
-    ]
+    summary_views = {
+        query_name: query
+        for query_name, query in summary_queries.items()
+        if query_name.endswith(view_suffix)
+    }
 
     if len(summary_views) < 1:
-        return view_name
+        return ()
+
+    summary_selects = [
+        f"{query_name} AS ({query})," for query_name, query in summary_views.items()
```

Comment:
```suggestion
        f"{query_name} AS ({query}) ," for query_name, query in summary_views.items()
```
Minor nit pick for making the query look nicer

Reply:
Done.